blockware/lib/VirtualBlox/stubs/utf8_decode.c

```diff
@@ -0,0 +1,193 @@
+/* utf8_decode.c */
+
+/* 2016-04-05 */
+
+/*
+Copyright (c) 2005 JSON.org
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+
+The Software shall be used for Good, not Evil.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+*/
+
+/* utf8_decode.h */
+
+#define UTF8_END   -1
+#define UTF8_ERROR -2
+
+extern int  utf8_decode_at_byte();
+extern int  utf8_decode_at_character();
+extern void utf8_decode_init(char p[], int length);
+extern int  utf8_decode_next();
```

Comment:
Including the header file content directly because otherwise the build was complaining it was undefined, absolutely no clue why but this fixed it so 🤷‍♂️

Reply:
like you couldn't `#include "utf8_decode.h"` here?